moveit_core/robot_model/src/robot_model.cpp

```diff
@@ -1347,27 +1348,29 @@ const LinkModel* RobotModel::getLinkModel(size_t index) const
 
 LinkModel* RobotModel::getLinkModel(const std::string& name, bool* has_link)
 {
-  if (has_link)
-    *has_link = true;  // Start out optimistic
   LinkModelMap::const_iterator it = link_model_map_.find(name);
   if (it != link_model_map_.end())
-    return it->second;
-
-  if (has_link)
   {
-    *has_link = false;  // Report failure via argument
```

Comment:
`has_link` was explicitly used to suppress the error message here and issue an error by the caller.